lib/improver/tests/set_up_test_cubes.py

```diff
@@ -72,13 +73,15 @@ def construct_xy_coords(ypoints, xpoints, spatial_grid):
             np.linspace(40, 80, xpoints, dtype=np.float32),
             "longitude", units="degrees")
     elif 'equalarea':
-        # use UK eastings and northings
+        # use UK eastings and northings on standard grid
         y_coord = DimCoord(
             np.linspace(-100000, 900000, ypoints, dtype=np.float32),
-            "projection_y_coordinate", units="metres")
+            "projection_y_coordinate", units="metres",
+            coord_system=STANDARD_GRID_CCRS)
```

Comment:
Thanks for adding this in the equalareas case. Can we add cartopy.crs.PlateCarree() for the latlon case please. There are other coordinate systems that use degrees as their coordinate unit but the values are quite different (though I guess you would call the vertical axis y rather than latitude) e.g. LambertCylindrical
```
pc = ccrs.PlateCarree()
lc = ccrs.LambertCylindrical()
lc.transform_point(100, 50, pc)
(100.0, 43.71187400016862)
```